### DISCUSSION TITLE
QNAP + Funkmodul - Wie?

### DISCUSSION THREAD

**Discussion by micha-dsk** [OpenCCU/OpenCCU#1182] [🇩🇪 Fragen&Antworten] [answered]
Ich bin dabei nach folgender Anleitung vorgegangen um RaspMatic als VM laufen zu haben (https://github.com/jens-maus/RaspberryMatic/wiki/Installation-QNAP-VirtualizationStation) Das läuft auch ganz wunderbar.

Nun würde ich gerne, bevor ich mir HomeMaticIP-Geräte kaufe, das Funkmodul zum laufen bekommen. Sehe ich es richtig, dass es reicht, folgende zwei Geräte + Gehäuse zu kaufen:

Platine: https://smartkram.de/produkt/platine-hb-rf-usb-tk/

Funkmodul: https://smartkram.de/produkt/neues-homematic-funkmodul-fuer-ccu3-fertig-aufgebaut-fuer-raspberrymatic-pivccu/

Gehäuse: https://smartkram.de/produkt/gehaeuse-fuer-hb-rf-usb-platine/

Wenn ich das alles zusammebaue (inkl. löten) und per USB an mein Nas anschließe sowie an die Virtuelle Maschine hänge, sollte ich doch Geräte anlernen können, oder?

Oder vergesse ich hierbei irgendetwas? Bitte um Hilfe :)

**Comment by jens-maus** [comment, marked as answer]
Das würde so gehen ja. Allerdings wäre es IMHO besser du besorgst dir folgende Komponenten:

1. `HB-RF-USB-2` Adapter-Platine:
   https://smartkram.de/produkt/platine-hb-rf-usb-2/
2. `RPI-RF-MOD` Funkmodul:
   https://de.elv.com/elv-komplettbausatz-funk-modulplatine-fuer-raspberry-pi-3-b-rpi-rf-mod-fuer-homematic-und-homematic-ip-152941
3. `ELV-Charly` Gehäuse:
    https://de.elv.com/elv-gehaeuse-rp-case-fuer-raspberry-pi-und-rpi-rf-mod-funk-modulplatine-schwarz-komplettbausatz-152970

Der Vorteil gegenüber deiner gewählten Auswahl wäre, dass das `HB-RF-USB-2` den exakten Formfaktor eines RaspberryPi hat und damit direkt in das ELV-Charly-Gehäuse samt `RPI-RF-MOD` Funkmodul passt.

Zu prüfen wäre allerdings ob deine QNAP auch überhaupt über diese Virtualization Station verfügt und dort RaspberryMatic funktioniert. Das könntest du ja vorab bereits ohne die oberen Komponenten prinzipiell testen.

**Reply by micha-dsk** [reply]
Hallo,

danke für die Antwort. Auf meiner QNAP läuft die RaspberryMatic bereits :) Danke, dann werde ich mir die benannten Teile einmal besorgen und weiter testen.
